plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/grouped-product-selector/style.scss

```diff
@@ -0,0 +1,5 @@
+.wp-block-woocommerce-add-to-cart-with-options {
+	.variations_form.cart {
```

Comment:
Do you know which component uses this selector `.wp-block-woocommerce-add-to-cart-with-options .variations_form.cart`? I wasn't able to find it either. :sweat: